src/sentry/models/relay.py

```diff
@@ -17,6 +17,7 @@ class RelayUsage(Model):
     version = models.CharField(max_length=32, default="0.0.1")
     first_seen = models.DateTimeField(default=timezone.now)
     last_seen = models.DateTimeField(default=timezone.now)
+    public_key = models.CharField(max_length=200, default="", db_index=True)
```

Comment:
This does not create a default in SQL, meaning that the deploy will break. Either, we need to enforce a DB default, or we need to make this nullable.

Reply:
While this is true, in practice I think we can live with it.

A registration error will happen for the cases when a new Relay tries to register exactly between the application of the migrations and the recycling of the web worker. 

For existing Relays that register the update should be fine even with the old model accessing the new schema.
So we only have a  small time when new relays are temporarily unable to register themselves.

Reply:
I'll leave this to your judgement, but do you even need a default of an empty string here? Could we just make this column nullable instead of a blank default?

Reply:
If we want to do this safely and keep the string default, my recommendation would be:
1. Change this column to be nullable with a default of "". Deploy
2. Add a new migration to make the column not null. I believe that Django will re-apply the default to the table to catch any other null values, but we can check the generated DDL to be sure in that pr.